react/components/ProductImages/index.js

```diff
@@ -38,23 +39,31 @@ const ProductImages = ({
     hiddenImages && hiddenImages.map(text => new RegExp(text, 'i'))
   const handles = useCssHandles(CSS_HANDLES)
 
-  const images = allImages
-    .filter(
-      image =>
-        !image.imageLabel ||
-        !excludeImageRegexes.some(regex => regex.test(image.imageLabel))
-    )
-    .map(image => ({
-      type: 'image',
-      url: image.imageUrls ? image.imageUrls[0] : image.imageUrl,
-      alt: image.imageText,
-      thumbUrl: image.thumbnailUrl || image.imageUrl,
-    }))
-  const videos = allVideos.map(video => ({
-    type: 'video',
-    src: video.videoUrl,
-    thumbWidth: 300,
-  }))
+  const images =
+    contentType !== 'videos'
```

Comment:
Just to make the code easier to understand, you could store this check in a variable, such as `shouldIncludeImages` and then this conditional would look a bit clearer:

```suggestion
    shouldIncludeImages
```